projects/core/directives/dropdown/dropdown-position-sided.directive.ts

```diff
@@ -46,7 +46,8 @@ export class TuiDropdownPositionSidedDirective extends TuiPositionAccessor {
         const {height, width} = rect;
         const hostRect = this.vertical.accessor?.getClientRect() ?? EMPTY_CLIENT_RECT;
         const viewport = this.viewport.getClientRect();
-        const {align, direction, minHeight, offset} = this.options;
+        const {direction, minHeight, offset} = this.options;
+        const align = this.options.align === 'center' ? 'left' : this.options.align;
```

Comment:
I imagine people would want sided center too 🤔 But we can probably leave it until somebody actually asks for it.

Reply:
I thought about it for a while and tuiDropdownAlign = 'center' in sided dropdown seems strange. May be tuiDropdownDirection = 'center' would be better for it, but will look strange for default dropdowns...